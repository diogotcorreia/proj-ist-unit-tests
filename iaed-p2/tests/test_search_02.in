set a 1
set b 2
set c 3
set d/a 5
set d 5
set b/a 6
set a/b 6
search 6
search 5
set b 6
search 6
set a/b/c/d/e/f/g/h/i 2
search 2
quit
