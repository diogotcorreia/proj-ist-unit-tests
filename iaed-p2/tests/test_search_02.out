/a/b
/d
/a/b
/a/b/c/d/e/f/g/h/i
